src/mlpack/methods/ann/layer/linear.hpp

```diff
@@ -52,6 +52,12 @@ class Linear
          const size_t outSize,
          RegularizerType regularizer = RegularizerType());
 
+  //! Copy constructor.
+  Linear(const Linear& layer);
+
+  //! Copy assignment operator.
+  Linear& operator = (const Linear& layer);
```

Comment:
```suggestion
  Linear& operator=(const Linear& layer);
```
Just a simple style fix. :)